### PR TITLE
Add script to upload files in folder

### PR DESCRIPTION
**Pull request by danleyb2**

### CODE REVIEW

File: stream/upload-folder.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+LOG_LEVEL = os.environ.get('LOGGING', 'INFO').upper()
+
+logging.basicConfig(
+    stream=sys.stdout,
+    level=LOG_LEVEL,
+    format='%(asctime)s %(levelname)s %(name)s %(threadName)s : %(message)s')
+
+
+def stream_api(i, args):
```
Comment (marcbelmont) [inline stream/upload-folder.py:16]: rename `i` to `image_path` for clarity

File: stream/upload-folder.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    if args.mask:
+        data = dict(mask_id=args.mask)
+    else:
+        data = None
+
+    with open(i, 'rb') as fp:
+        while True:
```
Comment (marcbelmont) [inline stream/upload-folder.py:23]: I don't understand why you're using a while loop here.

**Review by marcbelmont — COMMENTED** [review]
- rename the file to video_upload.py
- the output does not seem very usable. Is that what the client asked? Maybe saving the data in a jsonlines file would make more sense.

File: stream/upload-folder.py (hunk 2 above)
Comment (danleyb2) [reply stream/upload-folder.py]: This was based on the recognition_api method that retries **413**s.
It okay, Have also removed the check for `error` in response

**Comment by marcbelmont** [comment]
@danleyb2 see checkboxes in my previous comment above.

**Comment by marcbelmont** [comment]
> boxes in my

**Comment by danleyb2** [comment]
> Any update?

@marcbelmont this should be okay now, I am unable to mark the checkboxes though.

**Review by marcbelmont — APPROVED** [review]

**Pull request merged**